src/templates/init/test/test-file-info.ts

```diff
@@ -0,0 +1,15 @@
+interface testFileInfo {
+  fileName: string;
+  targetFolder: string;
+}
+
+export const testFiles: testFileInfo[] = [
```

Comment:
This should
- not be part of the `templates/` folder (actually it should not be necessary at all)
- be implicitly clear by the folder structure (I'm pretty sure `fs` and `path` have the functions you need)

Reply:
yes, used the basic reader method and model the strucutre right away